wiating_backend/elastic.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, name, description, directions, lat, lon, point_type, created_
         self.directions = directions
         self.lat = str(lat)
         self.lon = str(lon)
+        point_type_list = ['SHED', 'CABIN', 'CABIN_FIREPLACE', 'CABIN_HOSTED', 'TENT_CAMP', 'WATERFRONT', 'URBEX',
+                           'WATER_SOURCE', 'CAVE', 'SACRAL', 'TOWER', 'EMERGENCY']
+        if point_type not in point_type_list:
+            raise ValueError('point type is not on point type list')
```

Comment:
We need to handle this exception e.g. here https://github.com/wiating-app/wiating_backend/blob/master/wiating_backend/points.py#L31 and in all other places it can make an execution failure and return appropriate response to the user.